packages/extension/src/shared/transactions.ts

```diff
@@ -0,0 +1,20 @@
+import { Status } from "starknet"
+
+import { WalletAccount } from "./wallet.model"
+
+export interface TransactionMeta {
+  title?: string
+  subTitle?: string
+}
+
+export interface AddTransaction {
```

Comment:
class names and file names shouldn't start with a verb

```suggestion
export interface TransactionRequest {
```